store/src/server/api.rs

```diff
@@ -157,6 +158,8 @@ impl api_server::Api for StoreApi {
         }))
     }
 
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(skip(self), ret, err, fields(COMPONENT = crate::COMPONENT))]
     async fn get_transaction_inputs(
```

Comment:
Similar comments as the ones on the block producer. For the custom name, we could use `store::get_transaction_inputs` here.

Also, we probably shouldn't log full request/response at `INFO` level - but logging them at `DEBUG` level would probably be good.